src/Repository/InstalledPlugins.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * Installed plugins repository.
+ *
+ * @package SatisPress
+ * @license GPL-2.0-or-later
+ * @since 0.3.0
+ */
+
+declare ( strict_types = 1 );
+
+namespace SatisPress\Repository;
+
+use SatisPress\PackageFactory;
+use SatisPress\PackageType\Plugin;
+
+/**
+ * Installed plugins repository class.
+ *
+ * @since 0.3.0
+ */
+class InstalledPlugins extends AbstractRepository implements PackageRepository {
+	/**
+	 * Package factory.
+	 *
+	 * @var PackageFactory
+	 */
+	protected $factory;
+
+	/**
+	 * Create a repository.
+	 *
+	 * @since 0.3.0
+	 *
+	 * @param PackageFactory $factory Package factory.
+	 */
+	public function __construct( PackageFactory $factory ) {
+		$this->factory = $factory;
+	}
+
+	/**
+	 * Retrieve all installed plugins.
+	 *
+	 * @since 0.3.0
+	 *
+	 * @return array
+	 */
+	public function all(): array {
+		$items = [];
+
+		foreach ( get_plugins() as $plugin_file => $plugin_data ) {
```

Comment:
I guess support for mu_plugins could be added as a separate collection of classes now: `Repository\InstalledMuPlugins`, `PackageType\MuPlugin` etc.

May even just be able to get away with the Repository class, using `get_mu_plugins()` here, and use the same `build()` method.